android/src/ch/opengis/qfield/QFieldCameraPictureActivity.java

```diff
@@ -22,18 +22,28 @@
 public class QFieldCameraPictureActivity extends Activity{
     private static final String TAG = "QField Camera Picture Activity";
     private String prefix;
-    private String pictureFileName;
+    private String pictureFilePath;
+    private String suffix;
+    private String pictureTempFileName;
 
     @Override
     protected void onCreate(Bundle savedInstanceState){
         Log.d(TAG, "onCreate()");
         super.onCreate(savedInstanceState);
-
+        
         prefix = getIntent().getExtras().getString("prefix");
         Log.d(TAG, "Received prefix: " + prefix);
 
+        pictureFilePath = getIntent().getExtras().getString("pictureFilePath");
+        Log.d(TAG, "Received pictureFilePath: " + pictureFilePath);
```

Comment:
They can be useful when there are issues and the user sends us a catlog, no?

Reply:
If we think this to the end we could add a debug line between every line of code we ship. I'm not sure this will make QField faster and debugging easier.

I would only log where we reasonably expect issues. And then adding a single line with a high density of information. I.e. if issues are expected here, reduce it to a single debug line with all information in the end. If this was just useful during development and is no longer expected to give insights, remove it altogether.

Reply:
I will merge them a little bit. But I think it's not that bad to have the information here still.  Of course, maybe they can be improved. But this part of the code is something where issues happen with external applications or on specific devices and we cannot always reproduce it. Then it's nice to have a base of information still....

Reply:
Perfect :+1: